purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterBackendEvent.kt

```diff
@@ -0,0 +1,23 @@
+package com.revenuecat.purchases.customercenter.events
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CustomerCenterBackendEvent(
```

Comment:
this can be made `internal`

you would need to add `@get:JvmSynthetic ` to each property as well to hide it from Java, since Java doesn't recognize internal. By the way, if you can add that to `PaywallBackendEvent` as well (JvmSynthetic is missing there)